src/client/renderingengine.cpp

```diff
@@ -567,6 +568,7 @@ void RenderingEngine::draw_anaglyph_3d_mode(Camera *camera, bool show_hud, Hud *
 			irr::scene::ESNRP_TRANSPARENT +
 			irr::scene::ESNRP_TRANSPARENT_EFFECT + irr::scene::ESNRP_SHADOW;
 	camera->getCameraNode()->setPosition(rightEye);
```

Comment:
a suggestion, because this 3 calls are very repetitive, add a private function in renderingengine to handle this 3 calls directly